esmvaltool/config-reformat-file.yml

```diff
@@ -10,6 +10,6 @@ input_dir: ./TEST_INPUT
 # Destination directory
 output_dir: ./TEST_OUTPUT
```

Comment:
Can we use the original `config-file.yml` for this?

Reply:
What about using `rootpath` as a dedicated input-dir, just adding an extra project RAWOBS?
```
rootpath:
  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
  OBS: ~/obs_inputpath
  RAWOBS: ~/rawobs_inputpath
  default: ~/default_inputpath
```
And then we can add another command line flag to run the tool in reformat mode, as you suggest (I would prefer to call this cmorize mode, though).